samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -54,58 +54,48 @@ This sample is separated into a number of smaller functional units.
 
-1) Configures and activates the cellular modem.
-#) Starts a connection to the LTE network.
-#) Maintains a connection to `nRF Cloud`_.
-#) Re-establishes or retries all lost or failed connections.
-
-The modem only needs to be set up and connected to LTE once.
-If LTE is lost, the modem automatically works to re-establish it.
+Cloud connection loop
+=====================
 
-The connection to nRF Cloud must be maintained manually by the application.
-A significant portion of the :file:`src/connection.c` file is dedicated to this task.
+The cloud connection loop (implemented in :file:`src/cloud_connection.c`) monitors network availability, and starting a connection with `nRF Cloud`_ whenever the internet becomes reachable, and closing that connection whenever internet access is lost.
+It has error handling and timeout features to ensure failed or lost connections are re-established after a waiting period (:kconfig:option:`CONFIG_CLOUD_CONNECTION_RETRY_TIMEOUT_SECONDS`).
 
-This file also maintains a number of `Zephyr Events`_ tracking the state of the LTE and nRF Cloud connections.
-The connection management loop relies on these events to control its progression.
+LTE is enabled and connected to automatically by Zephyr's ``conn_mgr`` and :kconfig:option:`CONFIG_LTE_CONNECTIVITY`.
 
-The connection management loop must conform to the :ref:`nRF Cloud connection process <lib_nrf_cloud_connect>`.
-This is mostly handled by the :ref:`lib_nrf_cloud` library, with exception to the following behavior:
+Whenever a connection to nRF Cloud is started, the cloud connection loop follows the :ref:`nRF Cloud connection process <lib_nrf_cloud_connect>`.
+The :ref:`lib_nrf_cloud` library handles most of the connection process, with exception to the following behavior:
 When a device is first being associated with an nRF Cloud account, the `nRF Cloud MQTT API`_ will send a user association request notification to the device.
 Upon receiving this notification, the device must wait for a user association success notification, and then manually disconnect from and reconnect to nRF Cloud.
 Notifications of user association success are sent for every subsequent connection after this as well, so the device must only disconnect and reconnect if it previously received a user association request notification.
-This behavior is handled by the ``NRF_CLOUD_EVT_USER_ASSOCIATION_REQUEST`` and ``NRF_CLOUD_EVT_USER_ASSOCIATED`` cases inside the :c:func:`cloud_event_handler` function in the :file:`src/connection.c` file.
-
-The application must also manually reset its connection to nRF Cloud whenever LTE is lost.
-Otherwise, the :ref:`lib_nrf_cloud` library will deadlock.
-The ``LTE_LC_EVT_NW_REG_STATUS`` case inside the :c:func:`lte_event_handler` function performs this task.
+This behavior is handled by the ``NRF_CLOUD_EVT_USER_ASSOCIATION_REQUEST`` and ``NRF_CLOUD_EVT_USER_ASSOCIATED`` cases inside the :c:func:`cloud_event_handler` function.
```

Comment:
Do you really call them "cases". Are they not rather events or something like that?

Reply:
Yep -- The events are handled by the corresponding cases in a switch statement, so when referring to the event handler, it is better to say case